src/index.html

```diff
@@ -16,20 +16,32 @@
 					<label for="main-nav-button"><img class="nav-image" src="menu-alt-512.png"></label>
 					<input class="link-nav" type="checkbox" id="main-nav-button">
 					<nav class="main-nav">
-
 						<ul class="main-nav-list">
 							<li class="active"><a href="index.html">HOME</a></li>
 							<li><a href="#">WHAT WE DO</a></li>
 							<li><a href="#">WHY WE DO IT</a></li>
 							<li><a href="#">HOW WE DO IT</a></li>
 							<li><a href="#">JOIN US</a></li>
 							<li><a href="#">STANDARDS</a></li>
-							<li><a href="#">CONTACT</a></li>
+							<li><a><label for="contact-opening">CONTACT</label></a></li>
```

Comment:
Button will give a hover effect without being clickable on the sides of the button itself, leading to bad UX